File: library/src/main/java/name/zeno/android/third/glide/sizableurl/ZQiniuUrl.java
package name.zeno.android.third.glide.sizableurl;

/**
 * @author 陈治谋 (513500085@qq.com)
 * @since 2017/8/29
 */
public class ZQiniuUrl extends SizableUrlModel
{
  @Override public String requestCustomSizeUrl(int width, int height)
  {
    if (baseUrl == null) {
      return null;
    }
    return baseUrl + "?imageView2/2/w/" + width + "/h/" + height;
  }
}

File: library/src/main/java/name/zeno/android/third/glide/sizableurl/ZQiniuUrl.kt
package name.zeno.android.third.glide.sizableurl

/**
 * @author 陈治谋 (513500085@qq.com)
 * @since 2017/8/29
 */
class ZQiniuUrl : SizableUrlModel() {
  override fun requestCustomSizeUrl(width: Int, height: Int): String? = when (baseUrl) {
    null -> null
    else -> "$baseUrl?imageView2/2/w/$width/h/$height"
  }
}
